src/cpp_common/utilities.cpp

```
@@ -0,0 +1,129 @@
+ ********************************************************************PGR-GNU*/
+
+#include "cpp_common/utilities.hpp"
+
+#include <string>
+
+#include "c_common/enums.h"
+
+namespace pgrouting {
+
+std::string
+get_name(Which which) {
```

Cppcheck (2.19.0)

[style] The function 'get_name' is never used.

(unusedFunction)